src/components/savings/SavingsPredictionStepper.js

```diff
@@ -75,10 +80,12 @@ const SavingsPredictionStepper = ({ asset, balance, interestRate }) => {
       const formattedValue = handleSignificantDecimals(value, decimals, 1);
 
       return isSymbolStablecoin(symbol)
-        ? `$${formattedValue}`
+        ? nativeSelected?.alignment === 'left'
```

Comment:
same here, can use `convertAmountToNativeDisplay` from utilities.js and pass along the nativeCurrency itself